website/settings/defaults.py

```diff
@@ -1958,3 +1964,6 @@ class CeleryConfig:
 DS_METRICS_BASE_FOLDER = None
 REG_METRICS_OSF_TOKEN = None
 REG_METRICS_BASE_FOLDER = None
+
+PUBLIC_PROJECT_STORAGE_CAP = 50
```

Comment:
These don't appear to be used here; I expect they will be in a future iteration?

Reply:
Yeah. This was just a to-do list item for this ticket.